examples/devicefarm/testgridproject.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: devicefarm.aws.upbound.io/v1beta1
+kind: TestGridProject
+metadata:
+  annotations:
+    upjet.upbound.io/manual-intervention: "Requires manually add securityGroupIds and subnetIds"
```

Comment:
I believe we could remove this manual intervention annotation after adding references as I asked below.